components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -203,6 +203,21 @@ __attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
  */
 @property(nonatomic) UIImageRenderingMode imageRenderingMode;
 
+/**
+ Determines if a divider should be shown between the header and actions. To customize the divider
+ color see @c headerDividerColor.
+
+ Defaults to NO.
+ */
+@property(nonatomic, assign) BOOL showHeaderDivider;
```

Comment:
Based on the code, this only has any effect if there is either a title or message (or both).  Please update the documentation (or implementation) to reflect the correct behavior.

Reply:
I didn't really like the _magic_ I had added in that situation. I think its better just to have clients have full control over showing the headerDivider or not regardless of the title and message being set. This has been cleaned up.